src/debugAdapter/goDebug.ts

```diff
@@ -183,7 +187,7 @@ class Delve {
 				}
 			}
 			let dlvArgs = [mode || 'debug'];
-			if (mode === 'exec') {
+			if (mode === 'exec' || mode === 'debug') {
```

Comment:
Why is the program name being passed to `dlv debug`?  I don't believe it takes a program name as an argument.